jupyterhub/handlers/pages.py

```diff
@@ -172,7 +172,16 @@ def _render_form(self, for_user, spawner_options_form, message=''):
                 spawner._spawn_future = None
             # not running, no form. Trigger spawn and redirect back to /user/:name
             f = asyncio.ensure_future(self.spawn_single_user(user, server_name))
-            await asyncio.wait([f], timeout=1)
+            done, pending = await asyncio.wait([f], timeout=1)
```

Comment:
I thought [line 216](https://github.com/jupyterhub/jupyterhub/pull/2684/files#diff-f14530788c189c34e88032abd7c584e0R216) in post handler need rewrite in same way as well.